systers_portal/templates/meetup/snippets/meetup_sidebar.html

```diff
@@ -1,9 +1,14 @@
+{% if user.is_authenticated and user.is_active %}
 <div class="sidebar-module mb40">
   <h4>Meetup Location Actions</h4>
   <ol class="list-unstyled">
+    {% if meetup.slug %}
```

Comment:
This is not a Meetup Location Actions, but rather a Meetup Action.
Create it a separate sidebar under Meetup Location Actions. Same should be moved "Add New Meetup". And "Add" should come before "Delete"